modules/test/playwright/pages/site-admin-web/AccountsSiteSettingsPage.ts

```diff
@@ -0,0 +1,66 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {Locator, Page} from '@playwright/test';
+
+import {clickAndExpectToBeVisible} from '../../utils/clickAndExpectToBeVisible';
+import {PORTLET_URLS} from '../../utils/portletUrls';
+
+export class AccountsSiteSettingsPage {
+	readonly allowedAccountTypeControls: Locator;
+	readonly defaultValuesAlert: Locator;
+
+	constructor(private readonly page: Page) {
+		this.allowedAccountTypeControls = page.getByLabel(
+			'Allowed Account Types'
+		);
+		this.defaultValuesAlert = page
+			.getByRole('alert')
+			.first()
+			.getByText(
+				'Info:This configuration is not saved yet. The values shown are the default.'
+			);
+	}
+
+	async goto(siteUrl?: Site['friendlyUrlPath']) {
+		await this.page.goto(
+			`/group${siteUrl || '/guest'}${PORTLET_URLS.siteSettingsPlatformAccounts}`
+		);
+	}
+
+	async clickOnAction(actionName: string) {
+		await clickAndExpectToBeVisible({
+			autoClick: true,
+			target: this.page.getByRole('menuitem', {name: actionName}),
+			trigger: this.page.getByRole('button', {name: 'Actions'}),
+		});
+	}
+
+	async exportConfiguration() {
+		const downloadPromise = this.page.waitForEvent('download');
+
+		await this.clickOnAction('Export');
+
+		let fileContent = '';
+		const readStream = await (await downloadPromise).createReadStream();
+
+		for await (const chunk of readStream) {
+			fileContent += chunk.toString();
+		}
+
+		return fileContent;
+	}
```

Comment:
You could replace the complex stream consumption logic (for await...of) with the function `readFile`, there's an example [here](https://github.com/liferay/liferay-portal/blob/master/modules/test/playwright/tests/configuration-admin-web/main/instanceSettings.spec.ts#L64-L82). In my opinion, it makes the code more straightforward and clean. What do you think ?

```suggestion
	async exportConfiguration() {
		const downloadPromise = this.page.waitForEvent('download');

		await this.clickOnAction('Export');

		const download = await downloadPromise;

		const path = await download.path();

		const fileContent = await readFile(path, 'utf-8');

		return fileContent;
	}
```